site/viamguides/src/smart-snack-dispenser/smart-snack-dispenser.md

```diff
@@ -0,0 +1,444 @@
+   ![food dispenser without bowl](assets/hardwarePrepped.webp)
+1. Turn the dispenser upside down to access the screws on the bottom of the device.
+   ![flip the machine](assets/hardwareFlip.webp)
+1. Use the screwdriver to remove the screws in each corner. Set them aside for in a safe place for later when reassembling.
+   ![unscrew base](assets/hardwareUnscrew.webp)
+1. **Carefully**, flip the device over again with the food container (hopper) facing up. Slowly, lift the container to separate it from the base of the device and expose the internal wiring.
+   ![separate container from base](assets/hardwareDisassemble.webp)
+
+### Prepare hardware components
+
+1. Using the wire strippers, cut the wires connected to the motor and limit switch on the bottom of the food container.
+   **Make sure to keep around 6 inches of wire connected to each component.** This will make it easier to connect them to the other hardware peripherals later.
+   ![cut motor, switch wires](assets/hardwareComponentWires.webp)
+1. Using the wire strippers, cut the power (typically red) and ground (typically black) wires connected to power port at the bottom of the base.
+   **Make sure to keep around 6 inches of wire.** This will make it easier to connect them to the other hardware peripherals later.
+   ![power wires](assets/hardwarePowerWires.webp)
+1. For each of the cut wires, use the wire strippers to expose about half a fingernail's length of metal from the plastic covering. Take your time and pull slowly to remove only the plastic.
+   ![stripped wire](assets/hardwareWireStripped.webp)
+1. Set the food container part aside now that it is disconnected from the base. Place the base on its front so the back of the device is facing up.
+   Put on safety glasses and use the drill to slowly bore a hole in the base a couple inches from the bottom. Make sure you have a secure grip to prevent it from spinning while drilling. It should enter the base just above the power port.
+   ![camera cable hole](assets/hardwareCameraCableHole.webp)
+1. This hole will be used by the webcam's USB cable to connect to the Raspberry Pi once it is embedded in the food dispenser. Test it by pushing the USB end through several inches.
+   If there are any issues, use the drill to widen the entry point with a bigger drill bit or by slowly rotating the existing bit around the inside of the hole while the drill is running, applying light pressure to strip away material.
+   ![test camera entry](assets/hardwareCameraCableEntry.webp)
+1. Clean out any plastic material from the drilling process from the inside of the base and place the base with the bottom down to prepare for wiring.
+
+### Connect hardware components
+
+![wiring diagram](assets/wiring.webp)
+
+1. Use the above wiring diagram to connect the hardware peripherals and power together. The stripped wires from the snack dispenser can be connected to the jumper wires unless the WAGO lever nuts.
```

Comment:
I keep rereading `can be connected to the jumper wires unless the WAGO lever nuts`, but am unsure what that means 😅

Reply:
I meant to say "using" the WAGO lever nuts 🤦